src/main/webapp/app/shared/select/SelectOption.tsx

```diff
@@ -0,0 +1,32 @@
+import React from 'react';
+
+type SelectText = {
+  label: string;
+  text: string;
+};
+
+interface IEntitySelectOption {
+  title: string;
+  subTitles?: SelectText[];
+}
+
+export const EntitySelectOption: React.FunctionComponent<IEntitySelectOption> = props => {
```

Comment:
Does it still support search keyword highlight now?

Reply:
No, the search keyword highlighting is available in the EntitySearchOption component: https://github.com/calvinlu3/oncokb-transcript/blob/add-alias-to-gene-select/src/main/webapp/app/shared/search/SearchOptions.tsx#L12-L44.

Should we also highlight keyword when we're searching in the gene or drug selects for example?